openedx/frontend-component-header
# fix: studio dropdown hover styles

JIRA Ticket: [TNL-11205](https://2u-internal.atlassian.net/browse/TNL-11205)

>  Nav bar text disappears when focus and hover states are both true. Very interesting find! This will be true for all pages using the header from frontend-component-header. This does not happen when a user is only hovering over a button. This bug only appears when using the edx brand styles. This seems to be happening to all buttons that use the tertiary variant.

Testing

1. Using the tab key, tab over to any of the nav bar dropdowns.
2. Hover over the active dropdown with your mouse.
3. All text should remain visible with no button color change.
4. Hover over another button that is not focused.
5. The background of the button should be blue with white text.
